src/app/shell/workshop-details/side-menu/side-menu.component.ts

```diff
@@ -13,12 +14,24 @@ import { Observable } from 'rxjs';
 export class SideMenuComponent implements OnInit {
```

Comment:
you added ngOnDesctroy, then you need to add this interface here
```suggestion
export class SideMenuComponent implements OnInit, OnDestroy {
```